images/kube-api-proxy/pkg/proxy/handler.go

```diff
@@ -137,29 +157,45 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	if err != nil {
 		logger.Error(fmt.Sprintf("Error transforming request: %s", req.URL.String()), logutil.SlogErr(err))
 		http.Error(w, "can't rewrite request", http.StatusBadRequest)
+		// TODO add another more specific abstraction layer on top of MetricsProvider: h.metrics.incomingRewriteErrorInc()
```

Comment:
Create task for this TODO

Reply:
This TODO was implemented in ProxyMetrics in metrics.go